### ISSUE TITLE
Two issues?

### ISSUE THREAD

**Issue by Yourname942** [Sunconure11/dungeon-mobs#40]
1) Is it possible to remove the music discs dropping from destrachan? 
2) the spike traps fly up infinitely if you jump above them (please make them stop after a certain distance and/or make them go down after a certain amount of time)

**Comment by BlackJar72** [comment]
1. It was planned to make the drops configurable with JSON like in newer mods, but I've never gotten around to it as I've been slowly withdrawing from modding.

2. I had no idea they would even move up!

**Comment by Yourname942** [comment]
Yeah it's kind of funny to see, but also kind of immersion breaking. (If you are on a flat world it is easier to test)
